cave/com.raytheon.uf.viz.d2d.ui/plugin.xml

```diff
@@ -1452,6 +1452,24 @@
             schemeId="com.raytheon.viz.ui.cave.scheme"
             sequence="M1+H">
       </key>
+      <key
+            contextId="com.raytheon.uf.viz.d2d.ui"
+            commandId="com.raytheon.viz.ui.loadSerialized"
+            schemeId="com.raytheon.viz.ui.cave.scheme"
+            sequence="M1+1">
+      </key>
+       <key
+            contextId="com.raytheon.uf.viz.d2d.ui"
+            commandId="com.raytheon.viz.ui.saveProcedure"
+            schemeId="com.raytheon.viz.ui.cave.scheme"
+            sequence="M1+2">
+      </key>
+       <key
+            contextId="com.raytheon.uf.viz.d2d.ui"
+            commandId="com.raytheon.viz.ui.saveBundle"
+            schemeId="com.raytheon.viz.ui.cave.scheme"
+            sequence="M1+3">
```

Comment:
Change to `M1+M2+S` (Ctrl+Shift+S)

Before adding this hotkey, these keys asked to close an editor, but I didn't see where those hotkeys were defined in the code anywhere. After changing this hotkey it now opens the Save Bundle Locally GUI.